modules/gateways/monero/createinvoice.php

```diff
@@ -97,11 +101,23 @@ className: 'spinner', // The CSS class to assign to the spinner
             <div class='content-xmr-payment'>
             <div class='xmr-amount-send'>
             <span class='xmr-label'>Send:</span>
-            <div class='xmr-amount-box'>".$amount_xmr." XMR ($" . $amount . " " . $currency .") </div><div class='xmr-box'>XMR</div>
+            <div id='amount_xmr' value='$amount_xmr' class='xmr-amount-box'>$amount_xmr</div><div class='xmr-box'>XMR</div>
+            <button class='btn xmr-box-copy' data-clipboard-target='#amount_xmr'><img class='clippy' src='clippy.png' width='20' title='Copy to clipboard' alt='Copy to clipboard' /></button>
+            </div>
+            <br>
```

Comment:
I did not notice this before, maybe we could tweak a little bit the CSS for adding more space without using too many `<br>` elements.

Reply:
If you have any thoughts on what needs done let me know. Or feel free to edit before merging.